ibis/backends/bigquery/compiler.py

```diff
@@ -21,7 +21,6 @@
     reduction,
     unary,
 )
-from ibis.impala import compiler as impala_compiler
 from ibis.impala.compiler import ImpalaSelect, ImpalaTableSetFormatter
```

Comment:
Surely not in this PR, but those will eventually need to be moved and renamed. We'll want a `BaseSelect` in `base_sql` to be imported here, in spark... and impala will also need to import and subclass them.